src/test/java/com/codingnomads/betty/data/batch/batchprocessors/MatchToOddsProcessorTests.java

```diff
@@ -0,0 +1,45 @@
+package com.codingnomads.betty.data.batch.batchprocessors;
+
+import com.codingnomads.betty.data.models.FootballMatchInfo;
+import com.codingnomads.betty.data.models.MatchOdds;
+import com.codingnomads.betty.logic.services.GameInformationService;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class MatchToOddsProcessorTests {
+
+
+    private GameInformationService mockGameInformationService;
+    private MatchToOddsProcessor testMatchToOddsProcessor;
+    private MatchOdds mockMatchOdds;
+    private FootballMatchInfo mockFootballMatchInfo;
+    private List<FootballMatchInfo> matchInfoList;
+    private List<MatchOdds> matchOddsList;
+
+    @Before
+    public void setUp(){
+        mockGameInformationService = mock(GameInformationService.class);
+        testMatchToOddsProcessor = new MatchToOddsProcessor(mockGameInformationService);
+
+        mockMatchOdds = mock(MatchOdds.class);
+        mockFootballMatchInfo = mock(FootballMatchInfo.class);
+
+        matchInfoList = Arrays.asList(mockFootballMatchInfo);
```

Comment:
You should use a singleton list here to prevent accidental nullpointers. Consider changing 'Arrays.asList' to 'Collections.singletonList'